### ISSUE TITLE
Why are Fabric icons different to Fluent System icons?

### ISSUE THREAD

**Issue by Poopooracoocoo** [microsoft/fluentui-system-icons#470]
I wasn't sure where to put this but had to ask. Why are [Fabric](https://uifabricicons.azurewebsites.net/) icons or [Fluent UI](https://developer.microsoft.com/en-us/fluentui#/styles/web/icons) icons and [Fluent System](https://github.com/microsoft/fluentui-system-icons) icons so different to each other? Wouldn't it be normal to just have one set of icons across Microsoft products? Usage has been unclear: https://github.com/microsoft/fluentui/issues/12488. Google allows anyone to use Material Symbols.

What icons should a Fluent Design app use?

~~honestly there's so much going on in the fluent ui umbrella but i'm not here for that~~

**Comment by spencer-nelson** [comment]
This is mostly a documentation question related to FluentUI, not a Fluent System Icons issue. I'll try to connect the right people to further the discussion regarding usage of icons in FluentUI and how we message intent for Fluent as a system.

That being said, the intent for applications designed using the Fluent Design System is to use Fluent System Icons where possible. On the topic of an icon font, as discussed in the linked issue, it's something that mostly takes resources to support, which we've not really been given. Along those lines, most cases of iconography use are moving toward individual SVG files for display, as it's significantly simpler to handle in many ways, so that's what has been prioritized. If you have a particular need for fonts that isn't already supported, feel welcome to express your need so we can better track and prioritize, and especially feel welcome to fix or work with us to add said support.

**Comment by Poopooracoocoo** [comment]
I appreciate that response but why do they look so different in the first place? Are they both meant to be part of the Fluent Design System?

**Comment by Poopooracoocoo** [comment]
@spencer-nelson Could you follow up on your coworkers here?

**Comment by spencer-nelson** [comment]
Just to update, we are working on progress to help address this and provide further clarity. With Summer holidays a couple of key folks are/have been out on vacation but hope to give better resolution in ~ 2 weeks.

**Comment by YBAZAN** [comment]
Any news on this point?

**Comment by Poopooracoocoo** [comment]
@spencer-nelson Any updates on this?

**Comment by mdtauk** [comment]
> @spencer-nelson Any updates on this?

Isn't Fabric now Fluent 2, and isn't it now using the Fabric Icons?

![image](https://github.com/microsoft/fluentui-system-icons/assets/7389110/50b952d6-aa29-4518-b64b-1c26c2b45d46)

**Comment by Poopooracoocoo** [comment]
> > @spencer-nelson Any updates on this?
> 
> Isn't Fabric now Fluent 2, and isn't it now using the Fluent System Icons?

Yeah I know but I still think I'd rather see what Spencer can say Microsoft has truly achieved consistency across platforms with their icons and emoji, so I'll wait for them to close this issue.

**Comment by voidnothings** [comment]
I am also curious on the official take on this.
